Core.DomainServices/GDPR/DataProcessingRegistrationRemovedFromItContractEventHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Core.DomainModel.Events;
+using Core.DomainModel.GDPR.Events;
+
+namespace Core.DomainServices.GDPR
+{
+    internal class DataProcessingRegistrationRemovedFromItContractEventHandler : IDomainEventHandler<DataProcessingRegistrationRemovedFromItContractEvent>
```

Comment:
this is never invoked because you have not registered it. also notice that it is internal so it cannot be registered....

I will fix that